src/m365/spo/commands/listitem/listitem-get.ts

```diff
@@ -49,8 +49,8 @@ class SpoListItemGetCommand extends SpoCommand {
       `${args.options.webUrl}/_api/web/lists(guid'${encodeURIComponent(listIdArgument)}')`
       : `${args.options.webUrl}/_api/web/lists/getByTitle('${encodeURIComponent(listTitleArgument)}')`);
 
-    const fieldSelect: string = args.options.fields ?
-      `?$select=${encodeURIComponent(args.options.fields)}` :
+    const fieldSelect: string = args.options.properties ?
```

Comment:
Let's rename the const as well so that it's more intuitive and consistent with the new code